### DISCUSSION TITLE
Will g11n support Chinese?

### DISCUSSION THREAD

**Discussion by iblueer** [uxbox/uxbox#588] [General]
you can let us volunteers translate the ui elements.

**Comment by myfunnyandy** [comment]
Hi @iblueer, we would LOVE to have help to translate to chinese. Do you know anyone? Are you interested on doing it yourself? If so, let us know and we will help in anything needed.

Thanks!

**Reply by iblueer** [reply]
I‘d like to contribute for translation, but I am not good at Github operating. Should I fork the whole project and edit the locales.json file and send you a pull request?

**Reply by elhombretecla** [reply]
Thank you very much @iblueer for your help

You can do it through github.com directly. You just have to go to the [translation file](https://github.com/penpot/penpot/blob/develop/frontend/resources/locales.json) and edit it directly from there.

<img width="730" alt="edit-file" src="https://user-images.githubusercontent.com/1045247/108332619-4930c000-71d0-11eb-8902-3652cbcc58a7.png">

**Reply by iblueer** [reply]
Thank you for teaching. I want to add Simplified Chinese in it, what abbreviation should I take? zh_cn?

**Reply by elhombretecla** [reply]
Go for it!

**zh_nc** is perfect :)

**Reply by iblueer** [reply]
<img width="344" alt="image" src="https://user-images.githubusercontent.com/3581944/108333909-5f5c7180-720c-11eb-91d3-75d182eb3e63.png">
It looks a little weird, zh_cn right?

**Reply by elhombretecla** [reply]
sorry

**zh_cn**

**Reply by iblueer** [reply]
Roger that, Sir.

**Reply by elhombretecla** [reply]
You can start by translating a small piece, for example a screen, and send us the result for us to test. So we can check your contribution and see if the translations are working :)

**Reply by iblueer** [reply]
I translated some of those, please check: https://github.com/penpot/penpot/pull/665

**Comment by mirenbz** [comment]
Are the any plans, as @iblueer was asking, to allow contributed translations? Thanks!

**Reply by elhombretecla** [reply]
Hi!!! 
We are delighted to receive contributions in translations!
You can contribute by adding small / incremental patches to [this file](https://github.com/penpot/penpot/blob/develop/frontend/resources/locales.json)

If you have any questions, let us know :)
